index.js

```diff
@@ -0,0 +1,19 @@
+import HumanPlayer from "./humanPlayer.js"
+import Game from "./game.js"
+import View from "./view.js"
+
+
+// document.addEventListener("DOMContentLoaded",()=>{
+//     let button = document.createElement("button")
+//     button.innerText = "Let's Play!"
+//     document.body.appendChild(button)
+    
+document.addEventListener("DOMContentLoaded",()=>{
+    let el = document.querySelector("#game")
+    // let Human = new HumanPlayer("Human");
+    let game = new Game();
+    new View(el,game)
```

Comment:
Pay close attention to the argument order your View class takes in, as it takes in a game, then an el. But here you have them switched around.